flow/scripts/report_metrics.tcl

```diff
@@ -23,12 +23,12 @@ proc report_metrics { when {include_erc true} } {
     puts "\n=========================================================================="
     puts "$when max_slew_check_slack"
     puts "--------------------------------------------------------------------------"
-    sta::max_slew_check_slack
+    puts "[sta::max_slew_check_slack]"
```

Comment:
Would these messages go to stdout anyways?

Reply:
I assumed so, but I cannot find them in the log files. Example [here](https://content.jenkins.openroad.tools/static-files/boMnFEKor3QahpebnOWJeoSxWaZhAxAgwY4AZi3tmAwxNjUxMDYxNDA3MDM4OjEwOnZ2YmFuZGVpcmE6am9iL09wZW5ST0FELWZsb3ctc2NyaXB0cy1OaWdodGx5LVB1YmxpYy9sYXN0U3VjY2Vzc2Z1bEJ1aWxkL2FydGlmYWN0/flow/logs/nangate45/gcd/base/6_report.log).